### DISCUSSION TITLE
Drug File Schema

### DISCUSSION THREAD

**Discussion by db513** [CMSgov/price-transparency-guide#753] [General]
@shaselton-usds Are there any updates on the release of the updated schema and/or the enforcement of the rule regarding the drug files? We are now 9 months past the FAQ ending the enforcement delay.

**Comment by santaviccap** [comment]
@shaselton-usds seconding this!  Re-iterating that information will be released at a future data is not helpful.  Since deferral of enforcement of this rule has ended, does that technically mean the rule is being enforced right now and all organizations not posting is out of compliance?
